PrebidMobile/API1.0/src/test/java/org/prebid/mobile/PrebidServerAdapterTest.java

```diff
@@ -112,7 +115,13 @@ public void testInvalidPrebidServerAccountIdForRubiconHostedPrebidServer() {
         adapter.requestDemand(requestParams, mockListener, uuid);
         Robolectric.flushBackgroundThreadScheduler();
         Robolectric.flushForegroundThreadScheduler();
-        verify(mockListener).onDemandFailed(ResultCode.INVALID_ACCOUNT_ID, uuid);
+
+        ArgumentCaptor<ResultCode> resultCodeAC = ArgumentCaptor.forClass(ResultCode.class);
+        ArgumentCaptor<String> uuidAC = ArgumentCaptor.forClass(String.class);
+
+        verify(mockListener).onDemandFailed(resultCodeAC.capture(), uuidAC.capture());
+        assertThat(resultCodeAC.getValue(), Matchers.either(Matchers.is(ResultCode.INVALID_ACCOUNT_ID)).or(Matchers.is(ResultCode.PREBID_SERVER_ERROR)));
```

Comment:
What are the cases that enforce us to assert two ResultCode?
- INVALID_ACCOUNT_ID
- PREBID_SERVER_ERROR

This test case is for the INVALID_ACCOUNT_ID, then why should it pass if onDemandFailed is obtained with other ResultCode (PREBID_SERVER_ERROR in this case)

Also, can we mock the response instead? Something like this:
`server.enqueue(new MockResponse().setResponseCode(400).setBody("Invalid request format: No stored request found for id: 1001_INVALID_ACCOUNT_ID"));`